packages/cli/src/util/passphrase.ts

```diff
@@ -1,15 +1,13 @@
 import fs from "node:fs";
-import {getValidatorPassphrasePath} from "../validatorDir/paths.js";
-import {stripOffNewlines} from "./stripOffNewlines.js";
-import {writeFile600Perm} from "./fs.js";
 
 /**
  * Utility to read file as UTF8 and strip any trailing new lines.
  * All passphrase files must be read with this function
  */
 export function readPassphraseFile(passphraseFile: string): string {
   const data = fs.readFileSync(passphraseFile, "utf8");
-  const passphrase = stripOffNewlines(data);
+  // Remove trailing new lines '\n' or '\r' if any
+  const passphrase = data.replace(/[\n|\r]+$/g, "");
```

Comment:
what was the reason for changing this line?

I would recommend that we clean this up, either use `stripOffNewlines` here and get rid of the function altogether

https://github.com/ChainSafe/lodestar/blob/9f7b83037978ceaba49119b9506567078d12fdec/packages/cli/src/util/stripOffNewlines.ts#L4

Reply:
I don't remember to be honest

Reply:
let's clean this up https://github.com/ChainSafe/lodestar/pull/5283